testing/unit_testing/src/2_ranks/laplace_1d.cpp

```diff
@@ -85,15 +85,26 @@ config = {
     assertFileMatchesContent("out4.0.py", referenceOutput0);
     assertFileMatchesContent("out4.1.py", referenceOutput1);
 
-    InputReader::HDF5::FullDataset r("out4_c.h5");
-    ASSERT_TRUE(r.hasAttribute("timeStepNo"));
-    ASSERT_TRUE(r.hasAttribute("currentTime"));
-    std::vector<double> geometry, solution;
-    r.readDoubleVector("geometry", geometry);
-    r.readDoubleVector("solution", solution);
-    compareArray(geometry, {0, 0,   0, 0.8, 0,   0, 1.6, 0,   0, 2.4, 0,
-                            0, 2.4, 0, 0,   3.2, 0, 0,   4.0, 0, 0});
-    compareArray(solution, {1.0, 0.8, 0.6, 0.4, 0.4, 0.2, 0.0});
+    InputReader::HDF5::FullDataset r1("out4.0_p.h5");
+    InputReader::HDF5::FullDataset r2("out4.1_p.h5");
+    {
+      ASSERT_TRUE(r1.hasAttribute("timeStepNo"));
+      ASSERT_TRUE(r1.hasAttribute("currentTime"));
+      std::vector<double> geometry, solution;
+      r1.readDoubleVector("geometry", geometry);
+      r1.readDoubleVector("solution", solution);
+      compareArray(geometry, {0, 0, 0, 0.8, 0, 0, 1.6, 0, 0, 2.4, 0, 0});
+      compareArray(solution, {1.0, 0.8, 0.6, 0.4});
```

Comment:
Why did you remove the last two values here?

Reply:
dont get the question, its a different test now.

Reply:
The question referred to the previous commit, not to develop.